templates/layouts/main_layout.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>SenNet User Registration and Profile</title>
+    <title>SenNet User Registration and Profile</title>
 
-<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bootstrap/css/bootstrap.min.css') }}" />
-<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/app.css') }}">
-
-<script type="text/javascript" src="{{ url_for('static', filename='js/fontawesome-all.min.js') }}"></script>
-<script type="text/javascript" src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
-<script type="text/javascript" src="{{ url_for('static', filename='bootstrap/js/bootstrap.min.js') }}"></script>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bootstrap/css/bootstrap.min.css') }}" />
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/app.css') }}">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/toastify.min.css') }}">
 
+    <script type="text/javascript" src="{{ url_for('static', filename='js/fontawesome-all.min.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='bootstrap/js/bootstrap.min.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/toastify-js.js') }}"></script>
```

Comment:
We should only load the `toastify.min.css`, `toastify-js.js`, and the `members-download.js` (`main.js` before renaming) for Admin users. Just put them within the `{% if session['isAdmin'] %}  load the resources here  {% endif %}` block should be good. Won't hurt if loading them all the time, consider this as a performance tuning based on the actual needs.

Reply:
@yuanzhou  Various other places throughout the app might need these features, so same as above reason.